src/utils/__tests__/changelog.test.ts

```diff
@@ -0,0 +1,504 @@
+/* eslint-env jest */
+
+jest.mock('../githubApi.ts');
+import { getGitHubClient } from '../githubApi';
+jest.mock('../git');
+import { getChangesSince } from '../git';
+
+import { SimpleGit } from 'simple-git';
+
+import {
+  findChangeset,
+  removeChangeset,
+  prependChangeset,
+  generateChangesetFromGit,
+} from '../changelog';
+
+describe('findChangeset', () => {
+  const sampleChangeset = {
+    body: '- this is a test',
+    name: 'Version 1.0.0',
+  };
+
+  test.each([
+    [
+      'regular',
+      `# Changelog\n## ${sampleChangeset.name}\n${sampleChangeset.body}\n`,
+    ],
```

Comment:
What about prereleases? Can we add tests like this one to test them?

Reply:
Not sure I understand the case you want to cover with a prerelease version. The title search here has nothing specific towards versions.

Btw these are old tests that already existed so not touching them in this PR. GitHub shows them as brand new as I renamed the test file to align with the main module (changes -> changelog)